src/tools/rust-analyzer/crates/hir-ty/src/layout.rs

```diff
@@ -110,7 +110,7 @@ struct LayoutCx<'a> {
 impl<'a> LayoutCalculator for LayoutCx<'a> {
     type TargetDataLayoutRef = &'a TargetDataLayout;
 
-    fn delay_bug(&self, txt: String) {
+    fn delayed_bug(&self, txt: String) {
```

Comment:
CC @Veykril this will be fun to handle.

Reply:
That particular commit isn't that important, so if it's a pain I can just remove it.

Reply:
Isn't that only a problem when we do a sync while the changes haven't been pushed to crates io yet? fwiw we can change the autopublish time to be right before your usual sync times if you want

Reply:
It is, but it's easy to patch when syncing back, then undo that when we upgrade to the published version.